### PR TITLE
React select

### PR DESCRIPTION
**Pull request by Wolfe1**

### CODE REVIEW

File: src/Zoomba/Helpers/ReactSelect.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Similar to the Select class found in the Selenium Library:
+#   https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/support/select.py
+# Uses <div> tags instead of <select> & <option>
+# See the ReactHelpers/README.md for more information on the structure of React-Select components
```
Comment (Wolfe1) [inline src/Zoomba/Helpers/ReactSelect.py:4]: @PhilTandyMiller Need to change this reference

**Review by neiljhowell — APPROVED** [review]
I always suggest using pylint to add standardization to our python code. I see some missing module and function documentation, nothing major.

**Review by hoangx — APPROVED** [review]
other than the bot mentioning 2 for Unused import. Looks good.

**Comment by Wolfe1** [comment]
> other than the bot mentioning 2 for Unused import. Looks good.

@hoangx That alert is from a  month ago, that has been addressed

**Comment by Wolfe1** [comment]
Coverage loss is on me, not sure why my last pull request didn't complain about it.

**Pull request merged**